corehq/apps/app_manager/models.py

```diff
@@ -5611,7 +5611,6 @@ def reapply_overrides(self):
         for key, ref in self.logo_refs.items():
             mm = CommCareMultimedia.get(ref['m_id'])
             self.create_mapping(mm, ref['path'], save=False)
-        self.save()
```

Comment:
where does the save happen now?

Reply:
misread, reverting this. thanks

Reply:
actually made it explicit, https://github.com/dimagi/commcare-hq/pull/25010/commits/f1a835e864ae22060119cc0295689566b484e050